<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaultLocation</class>
 <widget class="QWidget" name="FaultLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>触发模式  [单次]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>显示模式  [罗盘]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>触发通道      [1]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>触发时长      [2]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保存当前波形</string>
    </property>
   </item>
  </widget>
  <widget class="QwtCompass" name="Compass" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <property name="lineWidth" stdset="0">
    <number>4</number>
   </property>
   <property name="origin" stdset="0">
    <double>270.000000000000000</double>
   </property>
   <property name="minScaleArc" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="maxScaleArc" stdset="0">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:red;}</string>
   </property>
   <property name="text">
    <string>通道1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:yellow;}</string>
   </property>
   <property name="text">
    <string>通道2</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>59</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>幅值</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_DB_2">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>DB</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>脉冲</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Pulse_2">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>59</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>幅值</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_DB_1">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>DB</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>脉冲</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Pulse_1">
      <property name="styleSheet">
       <string notr="true">QLabel {color:white;}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{border: none;;
                            background:transparent;}
                            QProgressBar::chunk{
                            background-color:skyblue;}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{border: none;;
                            background:transparent;}
                            QProgressBar::chunk{
                            background-color:skyblue;}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
